### ISSUE TITLE
`Liquid syntax error - Unknown tag 'seo' included` when serving jekyll with safe mode

### ISSUE THREAD

**Issue by joshbuker** [jekyll/minima#716]
When attempting to serve a jekyll site with minima in safe mode, I get the following error:

```
Liquid Exception: Liquid syntax error (/workspaces/gsd-tools/website/vendor/bundle/ruby/3.2.0/gems/minima-2.5.1/_includes/head.html line 5): Unknown tag 'seo' included in /workspaces/gsd-tools/website/vendor/bundle/ruby/3.2.0/gems/minima-2.5.1/_layouts/default.html
bundler: failed to load command: jekyll (/workspaces/gsd-tools/website/vendor/bundle/ruby/3.2.0/bin/jekyll)
/workspaces/gsd-tools/website/vendor/bundle/ruby/3.2.0/gems/liquid-4.0.4/lib/liquid/document.rb:23:in `unknown_tag': Liquid syntax error (/workspaces/gsd-tools/website/vendor/bundle/ruby/3.2.0/gems/minima-2.5.1/_includes/head.html line 5): Unknown tag 'seo' included  (Liquid::SyntaxError)
```

This error does not happen if I omit safe mode, although it appears that `_includes/custom-head.html` is being ignored, as well as setting the theme in `_config.yml`:

```yml
minima:
  theme: dark
```

Not sure what exactly is going on here, adding jekyll-seo-tag to the plugins does not appear to solve this.

Should be able to replicate the issue with: https://github.com/joshbuker/gsd-tools/tree/7dcc6aa2ed4331bbd5169e0579ba1ac961d26a1e/website

**Comment by joshbuker** [comment]
Ah, okay I figured this out after sanity checking my Gemfile.

If anyone else runs into this issue, it appears to be because the plugins minima relies on (`jekyll-seo-tag` and `jekyll-feed`) must be included both in your `_config.yml` and `Gemfile`.

`_config.yml`
```yml
plugins:
  - jekyll-seo-tag
  - jekyll-feed
```

`Gemfile`
```ruby
group :jekyll_plugins do
  gem 'jekyll-seo-tag'
  gem "jekyll-feed"
end
```

This doesn't appear to solve the `_includes/custom.html` or theme issues, but it does solve serving with `--safe` mode enabled.

**Comment by mxkpp** [comment]
Thanks @joshbuker for the tip.  Would love to be able to use dark theme in safe mode as well.

**Comment by joshbuker** [comment]
@mxkpp Per #717, you can enable dark theme by updating to pull from the main branch. Themes aren't supported yet and are to be released in 3.0.
